netmap/doc.go

```diff
@@ -1,34 +1,34 @@
 /*
-Netmap contract is a contract deployed in NeoFS sidechain.
+Package netmap contains implementation of the Netmap contract for NeoFS systems.
 
 Netmap contract stores and manages NeoFS network map, Storage node candidates
 and epoch number counter. In notary disabled environment, contract also stores
 a list of Inner Ring node keys.
 
-Contract notifications
+# Contract notifications
 
 AddPeer notification. This notification is produced when a Storage node sends
 a bootstrap request by invoking AddPeer method.
 
-  AddPeer
-    - name: nodeInfo
-      type: ByteArray
+	AddPeer
+	  - name: nodeInfo
+	    type: ByteArray
```

Comment:
Type is PublicKey:
https://github.com/nspcc-dev/neofs-contract/blob/f276e24137bc5b6a5d97fc36bf18f11e6a9ee19e/netmap/netmap_contract.go#L710.